vmaas_sync/vmaas_sync.py

```diff
@@ -31,6 +32,8 @@
 VMAAS_REPOS_ENDPOINT = "%s/api/v1/repos" % VMAAS_HOST
 ENABLE_RE_EVALUATION = strtobool(os.getenv("ENABLE_RE_EVALUATION", "YES"))
 ENABLE_REPO_BASED_RE_EVALUATION = strtobool(os.getenv("ENABLE_REPO_BASED_RE_EVALUATION", "NO"))
+DEFAULT_PAGE = 1
```

Comment:
Please remove this occurrence as well, as you've stated, there's no reason why we should start from other page than 1st.